drake-torch-melodic.dockerfile

```diff
@@ -51,7 +51,9 @@ RUN apt-get update && apt-get install -qy \
     ros-melodic-robot-state-publisher \
     ros-melodic-joint-state-publisher \
     ros-melodic-tf-conversions \
-    ros-melodic-rviz
+    ros-melodic-rviz \
+    ros-melodic-rqt \
```

Comment:
Should we be pegging these to a version?  We don't seem to do that for anything which I am curious if that is normal for ros- packages.

Reply:
it's only updated on average once every 4 months. we are already deep in deprecation hell and pegging them won't help with that before we get a stable noetic build.